### DISCUSSION TITLE
Run nanoFramework on AT91SAM9X35

### DISCUSSION THREAD

**Discussion by ghost** [nanoframework/Home#1122] [General]
Hi would it be possible to run the nanoFramework on AT91SAM9X35, i have a number of these modules and feel they have a lot of life left in them, seems the nanoFramework would be a good fit for them

**Comment by josesimoes** [comment]
Hey!

I don't have any port for ARM926. I can't see anything technical reason preventing this from being possible.

**Reply by ghost** [reply]
Hi thanks for the response, how would I go about getting it ported?

**Reply by josesimoes** [reply]
The best course of action is always to start from a "close" target. In this case, there is none and there is the aditional layer of adding a support for a new platform.
Here's the doc on how to do it for an STM32 target: https://docs.nanoframework.net/content/stm32/create-community-board.html

In case this is for a comercial project, know that there are professional services available for taks like this.

**Reply by ghost** [reply]
could you recommend any of them?

**Reply by josesimoes** [reply]
I guess one of the STM32 high ends like the STM32F769 would be the closest to your target.

**Reply by ghost** [reply]
hi thanks for the response, sorry as in professional service could you recommend any?

**Reply by alberk8** [reply]
@josesimoes is the man.

**Reply by ghost** [reply]
ah nice and quick thanks :), @josesimoes is this something we could discuss?

**Reply by josesimoes** [reply]
> ah nice and quick thanks :), @josesimoes is this something we could discuss?

Sure thing! 🙂 

https://www.nanoframework.net/partner-program/partners-eclo-solutions/

Please send me an email ( 👀 github profile)
